src/encoding/encoding.ts

```diff
@@ -10,7 +10,16 @@
  *  5. Binary blob should be used for binary data and string for strings
  *  */
 
-import * as msgpack from 'algo-msgpack-with-bigint';
+import {
+  encode as msgpackEncode,
```

Comment:
where are encode/decode coming from here? Are these newly created somewhere (I may find later in review)

Reply:
These are from the `algorand-msgpack` library: https://github.com/algorand/msgpack-javascript/blob/main/src/index.ts